src/ThingooConnector.cpp

```diff
@@ -0,0 +1,65 @@
+/***************************************************************
+   ThingooConnector is a library for the ESP8266/Arduino platform
+   to communicate with Thingoo API
+   Built for KNI Project "Thingoo" https://github.com/ThingooKNI/
+ **************************************************************/
+#include "ThingooConnector.h"
+
+ThingooConnector::ThingooConnector(const char *host)
+{
+	_host = String(host);
+	_host.replace("/", "");
+}
+
+void ThingooConnector::set_client_credentials(String client_id, String secret_key)
+{
+	_client_id = client_id;
+	_secret_key = secret_key;
+}
+
+void ThingooConnector::set_ssl_certificate_fingerprint(const char *fingerprint)
+{
+	_fingerprint = fingerprint;
+}
+
+const char *ThingooConnector::_get_token()
+{
+	std::unique_ptr<BearSSL::WiFiClientSecure > client(new BearSSL::WiFiClientSecure);
+	Serial.println("Sending HTTPS request");
```

Comment:
Library should print to Serial only important/error messages to avoid mess, remove this line